cf-ci-orchestration/Dockerfile

```diff
@@ -24,6 +24,10 @@ RUN true \
     git-core \
     bash-completion docker-bash-completion systemd-bash-completion \
     python-xml python-pyOpenSSL python-devel \
+    && curl -LO "https://releases.hashicorp.com/terraform/0.12.2/terraform_0.12.2_linux_amd64.zip" \
```

Comment:
When possible, changes to the Dockerfile should happen at the end in a new run step to leverage caching, and avoid rebuilding layers (faster pushes to docker hub and faster pulls from docker hub)

Reply:
Do you want a change on this?

Reply:
No it's fine, let's just keep it in mindfor future changes here